ViewGenerator/build/ViewGenerator.targets

```diff
@@ -49,15 +49,15 @@
 
   <Target Name="VG_CleanLegacyJSFiles" >
     <ItemGroup>
-      <JSFilesToDelete Include="**\*.js" Exclude="node_modules\**\*.js"/>
+      <JSFilesToDelete Include="**\*.js" Exclude="node_modules\**\*.js;$(CleanFilesExcludePath)**\*.js"/>
```

Comment:
This condition is problematic in cases where CleanFilesExcludePath is not defined.
I would keep this item like before and add another Item:

<JSFilesToDelete Condition="$(CleanFilesExcludePath) != ''" Remove="..."